spring2/src/main/java/com/bates/spring/UserController.java

```diff
@@ -0,0 +1,73 @@
+package com.bates.spring;
+
+
+import com.bates.spring.dto.UserDto;
+import main.java.com.bates.spring.model.Users;
+import main.java.com.bates.spring.model.request.UserRequest;
+import main.java.com.bates.spring.model.response.response.UserResponse;
+import com.bates.spring.services.UserService;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.beans.BeanUtils;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("users")
+public class UserController {
+
+    private final UserService userservice;
+
+    public UserController(UserService userservice){
+        this.userservice = userservice;
+    }
+
+    @GetMapping(path = "/allUsers")
```

Comment:
Look up RESTful naming conventions for endpoints. This is not RESTful. You want your URIs to be `{collection}/{individual}?{params}`. So get all users would just be GET `"/users"`